charts/datadog/values.yaml

```
@@ -386,6 +386,13 @@ datadog:
   # datadog.leaderLeaseDuration -- Set the lease time for leader election in second
   leaderLeaseDuration:  # 60
 
+  # datadog.leaderElectionResource -- Selects the default resource to use for leader election.
+  # Can be:
+  # * "lease" / "leases". Only supported in agent 7.47+
+  # * "configmap" / "confimaps".
+  # "" to automatically detect which one to use.
```

sorry one more nit, could we also put a `*` here?
```suggestion
  # * "" to automatically detect which one to use.
```